boringNotch/components/Notch/NotchHomeView.swift

```diff
@@ -73,9 +78,18 @@ struct NotchHomeView: View {
                             MarqueeText(musicManager.songTitle, font: .headline, nsFont: .headline, textColor: .white, frameWidth: geo.size.width)
                             MarqueeText(musicManager.artistName, font: .headline, nsFont: .headline, textColor: .gray, frameWidth: geo.size.width)
                                 .fontWeight(.medium)
+
+                            MusicSliderView(sliderValue: $sliderValue,
+                                            duration: musicManager.songDuration,
+                                            color: musicManager.avgColor,
+                                            dragging: $dragging) { newValue in
+                                musicManager.seekTrack(to: newValue)
```

Comment:
Overwrite sliderValue locally to even smoother experience

Reply:
I'm not sure what you mean

Reply:
sliderValue is dependent on polling. It gets the value from a function that fetches the current playing media. So it has some delay. if you explicitly overwrite it under `musicManager.seekTrack(to: newValue)` line, like `sliderValue = newValue` then it ensures there will be no jumping between values.

Reply:
Oh ok, I'm not sure this would change anything because sliderValue is also bound to value which gets updated as soon as the thumb of the slider changes. When the slider is released, it then runs the seekTrack callback function with newValue == sliderValue. 

The jumping that I tried to fix happened because the poll of the musicManager directly after the seek returns values for timestamp and elapsed time from before the seek takes effect. This resulted in the slider returning back to where it would be if the seek hadn't happened until the seek takes effect and updates the musicManager state, setting the thumb back to where the user dragged it. To fix this I added a small delay after releasing where dragging is true (keeping the sliderValue locked) until I assumed the musicManager had the updated values. However, I could use timestamp to check whether the local or polled value is correct as well, which would probably be the better way to do this. Did you experience jumps when testing it?

Reply:
I am experiencing this

# Preview

https://github.com/user-attachments/assets/432cc25e-8748-419e-9b70-d6616682ffed

Reply:
Can you try with the latest changes? This seems to be a duration issue but I can't replicate it

Reply:
It's working now. Good job. 🚀